configs/quadrotor_x.xml

```diff
@@ -1,4 +1,7 @@
 <model name="quadrotor_x">
+    <jsbsimbridge>
+        <jsb_script>motor_0_fail.xml</jsb_script>
+    </jsbsimbridge>
```

Comment:
Does this mean that the motor will always fail for the quadrotor_x model?

Reply:
Yes, in this case I included that change for the purpose of presenting a working solution. A final version would not include. The intent would be that the default behavior for all airframes is without the script.